### ISSUE TITLE
Offline MSDF Font Generator 

### ISSUE THREAD

**Issue by strangezakary** [Brainstronk/TinyEngine#12]
I would like the engine to support high quality font rendering if the user / app wants it. Therefore we will need some sort of tool that generates MSDF Font Atlas's offline so that we can load them into the engine.

**Comment by demetri** [comment]
I can look into this. Simplest possible method is to use STB to grab geometry and Chlumsky to generate MSDF, but could probably do our own version of the latter since it's IMO kind of inscrutable code.

Any requirements/expectations here?

**Comment by strangezakary** [comment]
Yep, im totally okay with using that stuff as there's no reason to parse TTF and get our own geometry, since stb_ttf does a good job at that. Only requirement is to make sure its a standalone tool, everything else is up to you.
